Source/WebKit/WebProcess/com.apple.WebProcess.sb.in

```diff
@@ -1785,19 +1785,41 @@
     io_registry_get_root_entry
     io_service_close))
 
-(define (allow-mach-exceptions)
+(define (allow-mach-exceptions operation)
 #if HAVE(HARDENED_MACH_EXCEPTIONS)
     (when (equal? (param "CPU") "arm64")
         (with-filter (require-not (webcontent-process-launched))
-            (allow mach-message-send (kernel-mig-routine task_register_hardened_exception_handler)))
-        (allow mach-message-send (kernel-mig-routine thread_adopt_exception_handler)))
+            (allow operation (kernel-mig-routine task_register_hardened_exception_handler)))
+        (allow operation (kernel-mig-routine thread_adopt_exception_handler)))
     (when (not (equal? (param "CPU") "arm64"))
-        (allow mach-message-send (kernel-mig-routine thread_set_exception_ports))))
+        (allow operation (kernel-mig-routine thread_set_exception_ports))))
 #else
-    (allow mach-message-send (kernel-mig-routine thread_set_exception_ports)))
+    (allow operation (kernel-mig-routine thread_set_exception_ports)))
 #endif
 
-(if (and (equal? (param "ENABLE_SANDBOX_MESSAGE_FILTER") "YES") (defined? 'mach-kernel-endpoint))
+(when (defined? 'syscall-mig)
```

Comment:
Do we need to check `ENABLE_SANDBOX_MESSAGE_FILTER`?

Reply:
That is a good point. In this case that is not required, since `syscall-mig` is not a message filter.

Thanks for reviewing!